frame/contracts/src/schedule.rs

```diff
@@ -84,6 +84,9 @@ pub struct Schedule<T: Config> {
 
 	/// The weights for each imported function a contract is allowed to call.
 	pub host_fn_weights: HostFnWeights<T>,
+
+	/// The additional weight for calling a function per local of that function.
+	pub call_per_local_cost: u32,
```

Comment:
Should be part of `InstructionWeights` as `call_per_local` just next to the other call related costs. Also, by putting it here the derived default impl sets it to `0`.